app/src/apiClient.js

```diff
@@ -64,22 +66,23 @@ export const getMoonData = async (month, year) => {
         .replace(' fill="transparent" ', 'fill="url(#image11)"')
         .replace("rgb(255,255,210)", "white"),
       AlexPhase: calculateMoonPhase(phase.lighting),
+      month: month,
```

Comment:
I would extract all this svg transform logic into a separate function which takes an svg element as a param, and call that function here instead.  Leaving it where it is makes the `return` statement harder to read.